war/src/main/less/modules/icons.less

```diff
@@ -89,11 +89,12 @@
 }
 
 .build-status-icon__wrapper {
-  display: inline-block;
+  display: inline-flex;
   position: relative;
 }
+
 .build-status-icon__outer {
-  display: block;
+  display: flex;
```

Comment:
This line break the build history UI in the left bottom for each job, see

![image](https://user-images.githubusercontent.com/40587651/149096998-c313b0ce-f695-4990-8dcd-15176fc53035.png)

Reply:
Heya, what browser/OS are you using? Thanks!

Reply:
I am using Chrome in Desktop with version 97.0.4692.71, thanks for reply :)

Reply:
Weird enough I can replicate that on my pipelines, but other kinds of job types, like freestyle or maven, are unaffected.

<details>
<summary>Pipeline Job</summary>

![](https://i.imgur.com/DmCYjuG.png)

</details>

<details>
<summary>Freestyle job</summary>

![](https://i.imgur.com/1FsqeqZ.png)

</details>

Reply:
Lots of stuff is duplicated in pipeline from core which has caused issues like this before

Reply:
Caused by Pipeline stage view plugin modifying a core class =/

![image](https://user-images.githubusercontent.com/21194782/150491671-98f20ad7-fc39-4bc1-a067-1824ddc2dd29.png)

Reply:
https://github.com/jenkinsci/pipeline-stage-view-plugin/pull/193